README.md

```diff
@@ -2,49 +2,60 @@
 
 **LVGL ported to Emscripten to be converted to JavaScript**
 
-The result looks like this https://lvgl.io/demos
+The result looks like this: [https://lvgl.io/demos](https://lvgl.io/demos)
 
 # How to get started
 
 ## Install SDL
 
-Downlad [SDL](https://www.libsdl.org/) (a graphics library to ope na window and handle the mouse). On Linux:
-1. Find the current version of SDL2: `apt-cache search libsdl2 (e.g. libsdl2-2.0-0)`
+Download [SDL](https://www.libsdl.org/) (a graphics library to open a window and handle the mouse). On Linux:
+
+1. Find the current version of SDL2: `apt-cache search libsdl2` (e.g., `libsdl2-2.0-0`)
 2. Install SDL2: `sudo apt-get install libsdl2-2.0-0` (replace with the found version)
-3. Install SDL2 development package: `sudo apt-get install libsdl2-dev`
+3. Install the SDL2 development package: `sudo apt-get install libsdl2-dev`
 4. If build essentials are not installed yet: `sudo apt-get install build-essential`
 
+---
 
 ## Install Emscripten SDK
-Download the [Emscripten SDK](https://kripken.github.io/emscripten-site/) and make sure it is in your PATH.
+
+Download the [Emscripten SDK](https://kripken.github.io/emscripten-site/) and make sure it is in your `PATH`.
 
 1. `git clone https://github.com/emscripten-core/emsdk.git`
-2. `cd <path-to-emsdk>`
-3. `git pull`
-4. `./emsdk install latest`
-5. `./emsdk activate latest`
-6. `source ./emsdk_env.sh`
+1. `cd <path-to-emsdk>`
+1. `git pull`
+1. `./emsdk install latest`
+1. `./emsdk activate latest`
+1. `source ./emsdk_env.sh`
+
+More info here: [Emscripten Downloads](https://kripken.github.io/emscripten-site/docs/getting_started/downloads.html)
 
-More info here: https://kripken.github.io/emscripten-site/docs/getting_started/downloads.html
+---
 
 ## Get the Emscripten-LVGL project
+
 1. Be sure you ran `. <path-to-emsdk>/emsdk_env.sh` to add EMSDK to `PATH`
-2. In any directoy: `git clone --recursive https://github.com/lvgl/lv_web_emscripten.git`
-3. `cd <path-to-emscripten>`
-4. `mkdir cmbuild`
-5. `cd cmbuild`
-6. `emcmake cmake ..`
-7. `emmake make -j4`
-5. A file called `index.html` will be generated. Run this in your browser.
+1. In any directory: `git clone --recursive https://github.com/lvgl/lv_web_emscripten.git`
+1. `cd <path-to-emscripten>`
```

Comment:
```suggestion
1. `cd lv_web_emscripten`
```

Reply:
I do the same thing with `cd emsdk`.
Additionally, I numbering in the first list to use "1." consistently.
I’ve amended the commit and force-pushed the changes.